packages/eslint-config/src/configs/rules/eslint-core.js

```diff
@@ -32,7 +32,7 @@ const rules = {
 
   // specify curly brace conventions for all control statements
   // https://eslint.org/docs/rules/curly
-  curly: ["error", "multi-line"], // multiline
+  curly: 0,
```

Comment:
we've been using strings instead of the numbers thus far
```suggestion
  curly: "off",
```

Reply:
I know .. but prettier plugin disables it this way. Not sure why they do that, but there's probably a reason. I did like this just to be compliant with that plugin

Reply:
but, I don't really care .. let's keep things consistent instead and use `"off"`

Reply:
removed it instead .. it's not recommended, so removing = disabling